test/built-ins/Map/prototype/getOrInsertComputed/check-state-after-callback-fn-throws.js

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2025 Daniel Minor. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-map.prototype.getorinsertcomputed
+description: |
+  Check state after callback function throws
+info: |
+  Map.prototype.getOrInsertComputed ( key , callbackfn )
+
+  ...
+
+  6. Let value be ? Call(callbackfn, key).
+  ...
+features: [upsert]
+---*/
+var map = new Map();
+map.set(0, 'zero');
+map.set(1, 'one');
+map.set(2, 'two');
+
+assert.throws(Error, function() {
+  map.getOrInsertComputed(3, function() {
+    throw new Error('throw in callback');
+  })
+});
+
+// Check the values after throwing in callbackfn.
+assert.sameValue(map.get(0), 'zero');
+assert.sameValue(map.get(1), 'one');
+assert.sameValue(map.get(2), 'two');
```

Comment:
These checks should also check the state of `3` after each call to `getOrInsertComputed`.

Reply:
I think you're still missing checks for `map.get(3)`

Reply:
I added a check below for `map.has(3)` to be false, is there a reason to use `get` instead?